libs/db/payload-types.ts

```
@@ -6,19 +6,77 @@
   auth: {
     users: UserAuthOperations;
   };
   collections: {
     users: User;
+    brands: Brand;
+    media: Media;
     'payload-locked-documents': PayloadLockedDocument;
     'payload-preferences': PayloadPreference;
     'payload-migrations': PayloadMigration;
   };
   collectionsJoins: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)